src/test/java/com/microsoft/sqlserver/jdbc/connection/RequestBoundaryMethodsTest.java

```diff
@@ -231,61 +229,59 @@ public void testOpenTransactions() throws SQLException {
+            try (Statement stmt1 = con.createStatement()) {
+                con.beginRequest();
+                try (Statement stmt = con.createStatement()) {
+                    try (ResultSet rs = stmt.executeQuery("SELECT 1")) {
+                        rs.next();
+                        assertEquals(1, rs.getInt(1));
+                        con.endRequest();
+
+                        assertTrue(!stmt1.isClosed(),
+                                "Statement created outside of beginRequest()/endRequest() block should not be closed.");
+                        assertTrue(stmt.isClosed(),
+                                "Statement created inside beginRequest()/endRequest() block should be closed after endRequest().");
+                        assertTrue(rs.isClosed(), "ResultSet should be closed after endRequest().");
                     }
                 }
+            }
 
-                // Multiple statements inside beginRequest()/endRequest() block
-                con.beginRequest();
-                try (Statement stmt = con.createStatement()) {
-                    TestUtils.dropTableIfExists(AbstractSQLGenerator.escapeIdentifier(tableName), stmt);
-                    stmt.executeUpdate(
-                            "CREATE TABLE " + AbstractSQLGenerator.escapeIdentifier(tableName) + " (col int)");
-                    try (PreparedStatement ps = con.prepareStatement(
-                            "INSERT INTO " + AbstractSQLGenerator.escapeIdentifier(tableName) + " values (?)")) {
-                        ps.setInt(1, 2);
-                        ps.executeUpdate();
-
-                        try (Statement stmt1 = con.createStatement(); ResultSet rs = stmt1
-                                .executeQuery("SELECT * FROM " + AbstractSQLGenerator.escapeIdentifier(tableName))) {
-                            rs.next();
-                            assertEquals(2, rs.getInt(1));
-                            TestUtils.dropTableIfExists(AbstractSQLGenerator.escapeIdentifier(tableName), stmt);
-
-                            try (CallableStatement cs = con.prepareCall("{call sp_server_info}")) {
-                                cs.execute();
-                                con.endRequest();
-
-                                assertTrue(stmt.isClosed());
-                                assertTrue(ps.isClosed());
-                                assertTrue(stmt1.isClosed());
-                                assertTrue(cs.isClosed());
-                                assertTrue(rs.isClosed());
-                            }
+            // Multiple statements inside beginRequest()/endRequest() block
+            con.beginRequest();
+            try (Statement stmt = con.createStatement()) {
+                TestUtils.dropTableIfExists(AbstractSQLGenerator.escapeIdentifier(tableName), stmt);
+                stmt.executeUpdate(
+                        "CREATE TABLE " + AbstractSQLGenerator.escapeIdentifier(tableName) + " (col int)");

```

Comment:
Can fit on one line, please combine.
```suggestion
                stmt.executeUpdate("CREATE TABLE " + AbstractSQLGenerator.escapeIdentifier(tableName) + " (col int)");
```